lib/bundler/vendor/postit/lib/postit/installer.rb

```diff
@@ -19,6 +19,7 @@ def installed?
 
     def install!
       return if installed?
+      puts "Installing locked Bundler version #{@bundler_version}"
```

Comment:
please move this into the postit trampoline, vendored code shouldn't be modified directly